doc/md/CLI/welcome.md

```diff
@@ -0,0 +1,40 @@
+---
+id: Welcome to Atlas
+slug: /CLI/
+sidebar_position: 2
+---
+
+## CLI
+
+Work with any data source from the command line.
```

Comment:
For now at least, this is an incorrect claim, and i think we chose "a database toolkit" as the tag line for now until we come up with something better.

Reply:
Removed, will update the atlas docs.